ext/standard/image.c

```diff
@@ -5,7 +5,7 @@
    | This source file is subject to version 3.01 of the PHP license,      |
    | that is bundled with this package in the file LICENSE, and is        |
    | available through the world-wide-web at the following url:           |
-   | https://www.php.net/license/3_01.txt                                 |
+   | https://www.php.net/license/3_01.txt                                  |
```

Comment:
Spurious space addition in this file and the next.

Reply:
I can't believe I did that... now they should be gone.